koap/build.gradle.kts

```diff
@@ -32,7 +33,7 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 api(kotlin("stdlib"))
-                implementation("com.squareup.okio:okio-multiplatform:2.6.0")
+                implementation("com.squareup.okio:okio-multiplatform:2.9.0")
```

Comment:
Change unrelated?

Reply:
We weren't sure if some of the npm dependency packaging issues were plugin related or specific to our dependencies. The version bump was hoping that older okio version weren't properly configured for multiplatform. I don't believe that ended up being the case; so this version bump was unnecessary (for the purposes of this PR).

Reply:
So should we undo this?

Reply:
It is outside the scope of this PR. For me it's a `nit`, I'm fine with leaving it in as we should upgrade dependencies. If it's not too much trouble, then removing it from this PR is better.

Reply:
Make sense. Given this repo is very much in a pre-beta state I'll go ahead and leave it in.